README.md

```diff
@@ -74,3 +74,109 @@ ddb.getAsync(getParams)
   .then((result) => console.log('success', result))
   .catch((err) => console.log('error', err))
 ```
+
+### SNS
+
+To get an instance of the SNS wrapper:
+
+```js
+const sns = require('@sparkpost/aws').SNS({ 
+  account: 'abc123', 
+  arnSuffix: '-prd', 
```

Comment:
include the prefix in config as well? or call out that we prefix it with `sns-` in the readme.